app/index.html

```diff
@@ -1,21 +1,26 @@
 <!DOCTYPE HTML>
+<!--suppress HtmlUnknownTag -->
```

Comment:
What are these `<!--suppress` for?

Reply:
Suppress and noinspection comments are just inpections suppresser for IDE, they don't have a value in code, and if you use other environment or think that they are unwanted I get get rid of it